### ISSUE TITLE
DeepSeek-V3 sft loss

### ISSUE THREAD

**Issue by liaoning97** [alibaba/Pai-Megatron-Patch#537]
Hi, 
    Thank you for your open-source contribution on DPSK-V3 training. I have tried using your codebase to pretrain and finetune the V3 model, but the loss are quite similar. However, the finetuned loss is supposed to be much lower than the pretraining one. I have carefully checked that the converted mcore weights from hf(cast to fp16) has been loaded. I wonder if you have met the same problem.
    Hope for your reply. Thanks a lot!
